pages/disputes/index.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <h1>MESES DE DISPUTA</h1>
+
+    <v-btn
+      class="mt-10"
+      color="primary"
+      to="/disputes/new"
+    >
+      NOVO MÊS DE DISPUTA
+    </v-btn>
+
+    <DisputeCard
+      v-for="(dispute, index) in disputes"
+      :key="index"
+      :dispute="dispute"
+    />
+
+  </div>
+</template>
+
+<script>
+import DisputeCard from '@/components/disputes/DisputeCard'
+import gql from 'graphql-tag'
+
+export default {
+  middleware: ['auth'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
  middleware: ["auth"],
```